### PR TITLE
fix(output_workflow): show org slug instead of org id in output

### PR DESCRIPTION
**Pull request by j-luong**

### CODE REVIEW

File: pkg/configuration/constants.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ const (
 	INTEGRATION_ENVIRONMENT_VERSION string = "snyk_integration_environment_version"
 	ANALYTICS_DISABLED              string = "snyk_disable_analytics"
 	ORGANIZATION                    string = "org"
+	ORGANIZATION_SLUG               string = "org_slug"
```
Comment (PeterSchafer) [inline pkg/configuration/constants.go:16]: Suggestion: add `internal_` as we are not expecting this to be a explicit externally set value.

File: pkg/app/app.go [hunk 2]
```diff
@@ -38,6 +38,22 @@ func defaultFunc_FF_CODE_CONSISTENT_IGNORES(engine workflow.Engine, config confi
 	return callback
 }
 
+func defaultFuncOrganizationSlug(engine workflow.Engine, config configuration.Configuration, apiClient api.ApiClient, logger *zerolog.Logger) configuration.DefaultValueFunction {
+	callback := func(existingValue interface{}) interface{} {
+		client := engine.GetNetworkAccess().GetHttpClient()
+		url := config.GetString(configuration.API_URL)
+		apiClient.Init(url, client)
+		orgId := config.Get(configuration.ORGANIZATION)
```
Comment (PeterSchafer) [inline pkg/app/app.go:46]: Suggestion: better use `config.GetString()`

Comment (PeterSchafer) [reply pkg/app/app.go:46]: Also maybe add a case to handle an empty `orgId`

File: internal/api/contract/GetOrgResponse.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package contract
```
Comment (PeterSchafer) [inline internal/api/contract/GetOrgResponse.go:1]: Question: Does it make sense to generate this contract rather than manually maintaining it?

Comment (PeterSchafer) [reply internal/api/contract/GetOrgResponse.go:1]: We will postpone this topic a bit until we have a solution we can apply consistently.

Comment (j-luong) [reply internal/api/contract/GetOrgResponse.go:1]: 💯, but this sounds like another initiative which shouldn't block this PR

**Review by PeterSchafer — APPROVED** [review]

**Pull request merged**